server/notification-providers/smtp.js

```diff
@@ -4,21 +4,19 @@ const { DOWN } = require("../../src/util");
 const { Liquid } = require("liquidjs");
 
 class SMTP extends NotificationProvider {
-
     name = "smtp";
 
     /**
      * @inheritdoc
      */
     async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
-
```

Comment:
could you please limit yourself to only related code-style changes?
Like this, you are making our job unnecessarily harder.